<?xml version="1.0"?>

<launch>
  <arg name="input/laserscan" default="virtual_scan/laserscan" />
  <arg name="input/obstacle_pointcloud" default="no_ground/pointcloud" />
  <arg name="input/raw_pointcloud" default="concatenated/pointcloud" />
  <arg name="output" default="occupancy_grid" />
  <arg name="manager" />

  <!-- run pointcloud_to_laserscan node -->
  <node pkg="nodelet" type="nodelet" name="laserscan_to_occupancy_grid_map" args="load laserscan_to_occupancy_grid_map/laserscan_to_occupancy_grid_map_nodelet $(arg manager)">
    <remap from="~input/laserscan" to="$(arg input/laserscan)" />
    <remap from="~input/obstacle_pointcloud" to="$(arg input/obstacle_pointcloud)" />
    <remap from="~input/raw_pointcloud" to="$(arg input/raw_pointcloud)" />
    <remap from="~output/occupancy_grid_map" to="$(arg output)" />
    <param name="map_resolution" value="0.5" />
    <param name="use_height_filter" value="true" />
  </node>
</launch>
